templates/default/nginx-proxy.conf.erb

```diff
@@ -19,7 +19,6 @@ server {
 
   # upload service file chunks are 100mb
   client_body_buffer_size 101m;
-  client_body_temp_path <%= @body_temp_path %>;
 
```

Comment:
@lbjay do you recall where that "100mb" upload service file chunk size is configured? Is it set by us within another recipe file? I ask because it seems fragile to remove the fallback when that buffer and the upload chunk size are so tightly coupled.

Reply:
On the other hand, if the system chokes becaues nginx tries to use temp strorage again, it's a heads up to recalibrate those sizes.